src/app/models/data/Annotation.ts

```diff
@@ -37,12 +41,17 @@ export class Annotation extends AbstractModelWithoutId implements IAnnotation {
   public readonly deleterId: number;
   public readonly deletedAt: string | DateTimeTimezone;
   @bawSubModelCollection(Tag)
-  public readonly tags: Tag[];
+  public readonly unsortedTags: Tag[];
   public readonly audioRecording: AudioRecording;
   public readonly score: number;
   public readonly durationSeconds: number;
   public readonly channel: number;
   public readonly audioEventImportFileId: number;
+  public readonly tagComparer: TagComparer;
+
+  public get tags(): Tag[] {
+    return this.unsortedTags.sort(this.tagComparer);
+  }
```

Comment:
i'm guessing you went with the getter because if the search parameters change and this is read again due to the magic of the comparer closure, this returns a new sort?

Do we ever really need this though?

I'd have sorted the tags that were passed into the creation of this object, or sorted them on assignment with a setter

Reply:
I think you're right about just sorting them upon assignment